exchanges/kucoin/kucoin.go

```diff
@@ -26,119 +25,31 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
 	"github.com/thrasher-corp/gocryptotrader/exchanges/orderbook"
 	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+	"github.com/thrasher-corp/gocryptotrader/types"
 )
 
 // Kucoin is the overarching type across this package
 type Kucoin struct {
 	exchange.Base
 	obm *orderbookManager
+
+	// HFSpot whether a high-frequency spot order is enabled or not.
+	HFSpot bool
+	// HFMargin whether a high-frequency margin order is enabled or not.
+	HFMargin bool
```

Comment:
I'm not seeing anywhere where this is set. Is there an API endpoint that can do this? eg GetAccountInfo? It would be good if this is set automatically rather than rely on the user calling `k.HFSpot = true`

Reply:
I couldn't get a clear endpoint telling me whether the account is a high-frequency spot/margin account, so I left these variables like this. So, should I keep it like this or completely remove it?